dist/backdraft.js

```diff
@@ -532,6 +532,16 @@ _.extend(Plugin.factory, {
       return !columnConfig.bulk;
     });
 
+    // read initial filters from URL
+    var urlParamString = window.location.href.split("?")[1];
+    if (urlParamString && $.deparam(urlParamString) && $.deparam(urlParamString).ext_filter_json ) {
```

Comment:
I would suggest we create a small tech item for the future, once we don't use old reports, and completely replace ext_filter_json with a small logic for convert `ext_filter_json` to `filter_json`? It would be a half day item and that way this story makes it into QA by today